<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="FileAvailable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>256</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Gagnant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Player1"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Perdant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Player2"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Send">
       <property name="text">
        <string>Send Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="Classement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>256</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Nouveau ELO : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ELOP1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ELOP2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>141</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>ELO du joueur selectioné : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ELOSelected">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>131</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>ELO Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Max">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>ELO Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Min">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>ELO Moy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Moy">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEntrer_R_sultat">
   <property name="text">
    <string>Entrer Résultat</string>
   </property>
  </action>
  <action name="actionClassement">
   <property name="text">
    <string>Classement</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
